test/all_functional.spec.ts

```diff
@@ -10,12 +10,36 @@ const IGNORED_MODULES = new Set([
   '_defaultRefDate',
 ]);
 
-function isTestableModule(mod: string) {
-  return !IGNORED_MODULES.has(mod);
+function getMethodNamesByModules(faker: Faker): { [module: string]: string[] } {
+  return Object.fromEntries(
+    Object.keys(faker)
+      .filter(isTestableModule)
+      .sort()
+      .map<[string, string[]]>((moduleName) => [
+        moduleName,
+        getMethodNamesOf(faker[moduleName]),
+      ])
+      .filter(([module, methods]) => {
+        if (methods.length === 0) {
+          console.log(`Skipping ${module} - No testable methods`);
+          return false;
+        }
+
+        return true;
+      })
+  );
+}
+
+function isTestableModule(moduleName: string): moduleName is keyof Faker {
+  return !IGNORED_MODULES.has(moduleName);
+}
+
+function getMethodNamesOf(module: Faker[keyof Faker]): string[] {
+  return Object.keys(module).filter(isMethodOf(module));
 }
 
-function isMethodOf(mod: string) {
-  return (meth: string) => typeof fakerEN[mod][meth] === 'function';
+function isMethodOf(module: Faker[keyof Faker]): (method: string) => boolean {
```

Comment:
Nitpick: The "module" technically could be any object, which allows the function to be used in a more generic context.

```suggestion
function isMethodOf(module: object): (method: string) => boolean {
```